src/components/Avatar/AvatarCard.test-int.js

```diff
@@ -0,0 +1,135 @@
+/* global Feature, Scenario  */
+const searchWithinIframe = require('../../../test/codecept/utils/searchWithinIframe.js');
```

Comment:
Again, consider removing if we decide to include this in the `/test/codecept/steps_file.js`